### ISSUE TITLE
Checking if pyload throws a exception

### ISSUE THREAD

**Issue by Tockra** [pyload/pyload#3242]
Hi,

I recognized that when pyload throw an exception you can't use it anymore and need to restart it with console.
Is there a way to send a request to pyload to check if it's running still perfect?

T

**Comment by GammaC0de** [comment]
> I recognized that when pyload throw an exception you can't use it anymore

This should not happen even when exception occurs.

Please give an example for such event.

**Comment by Tockra** [comment]
I promise I had that situation already and the problem was that pyload didn't exit itself, so my dockercontainer didn't restart.  Because that I want to send some kind of request to pyload to check its health-state.
I don't know what I did to produce that failor, but it was there.

**Comment by GammaC0de** [comment]
I suggest you manualy build a new docker from https://github.com/obi12341/docker-pyload because the "latest" build at https://hub.docker.com/r/writl/pyload/builds/ is 5 month old..

**Comment by GammaC0de** [comment]
> I want to send some kind of request to pyload to check its health-state

That depends , You can try to send a HTTP request to web interface.

**Comment by GammaC0de** [comment]
Can this be closed?

**Comment by Tockra** [comment]
When you name me the request ;)

**Comment by GammaC0de** [comment]
> When you name me the request ;)

What !?

**Comment by Tockra** [comment]
> You can try to send a HTTP request to web interface

**Comment by Velociraptor85** [comment]
well just http request the webinterface if you get an timeout or an error i guess it stopped :? or do I get something wrong here ?

**Comment by Tockra** [comment]
Yes because the webinterface is still reachable wen pyload Crashs. It just not work
